waspc/stack.yaml

```diff
@@ -42,6 +42,7 @@ extra-deps:
   - strong-path-1.0.0.0
   - path-0.9.0
   - path-io-1.6.3
+  - dir-traverse-0.2.3.0
```

Comment:
Why is this here and not in package.yaml?

By the way, maybe you could use https://github.com/wasp-lang/wasp/blob/master/waspc/src/Util/IO.hs#L27 instead, so we don't have to add this extra dependency?

Reply:
This is exactly what I needed! Thanks you, will update properly.